src/Raven.Server/ServerWide/Commands/PutCertificateWithSamePinningHashCommand.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System.Collections.Generic;
+using Raven.Client.ServerWide.Operations.Certificates;
+using Raven.Server.ServerWide.Context;
+using Sparrow.Json.Parsing;
+
+namespace Raven.Server.ServerWide.Commands
+{
+    public class PutCertificateWithSamePinningHashCommand : PutCertificateCommand
+    {
+        public PutCertificateWithSamePinningHashCommand()
+        {
+            // for deserialization
+        }
+
+        public PutCertificateWithSamePinningHashCommand(string name, CertificateDefinition value) : base(name, value)
+        {
+
+        }
+
+        public override DynamicJsonValue ValueToJson()
+        {
+            return Value?.ToJson();
+        }
+
+        public override void VerifyCanExecuteCommand(ServerStore store, TransactionOperationContext context, bool isClusterAdmin)
+        {
+            if (Value.SecurityClearance == SecurityClearance.ClusterAdmin || Value.SecurityClearance != SecurityClearance.ClusterNode)
```

Comment:
I don't understand this logic? You are either a cluster admin or not a cluster node?

Reply:
I don't get it either, seems strange (it was copied from PutCertificateCommand). Fixing both.